.github/workflows/smoke-test-daily.yml

```diff
@@ -1,99 +1,173 @@
                   )
+              working-directory: plugins/woocommerce
+              run: pnpm exec allure generate --clean api-test-report/allure-results --output api-test-report/allure-report
+
+            - name: Archive API test report
+              if: |
+                  always() &&
+                  steps.generate_api_report.conclusion == 'success'
               uses: actions/upload-artifact@v3
               with:
-                  name: api-test-report---daily
+                  name: ${{ env.API_ARTIFACT }}
                   path: |
-                      ${{ env.API_TEST_REPORT_DIR }}/allure-results
-                      ${{ env.API_TEST_REPORT_DIR }}/allure-report
+                      plugins/woocommerce/api-test-report/allure-results
+                      plugins/woocommerce/api-test-report/allure-report
+                  if-no-files-found: ignore
                   retention-days: 5
 
+    k6-tests:
+        name: k6 tests on trunk
+        runs-on: ubuntu-20.04
+        needs: [api-tests]
+        if: always()
+        steps:
+            - uses: actions/checkout@v3
+              with:
+                  ref: ${{ env.BRANCH_NAME }}
+
+            - name: Setup WooCommerce Monorepo
+              uses: ./.github/actions/setup-woocommerce-monorepo
+              with:
+                  install-filters: woocommerce
+                  build: false
+
+            - name: Download and install Chromium browser.
+              working-directory: plugins/woocommerce
+              run: pnpm exec playwright install chromium
+
             - name: Update performance test site with E2E test
               if: always()
               working-directory: plugins/woocommerce
               env:
-                SMOKE_TEST_URL: ${{ secrets.SMOKE_TEST_PERF_URL }}/
-                SMOKE_TEST_ADMIN_USER: ${{ secrets.SMOKE_TEST_PERF_ADMIN_USER }}
-                SMOKE_TEST_ADMIN_PASSWORD: ${{ secrets.SMOKE_TEST_PERF_ADMIN_PASSWORD }}
-                SMOKE_TEST_ADMIN_USER_EMAIL: ${{ secrets.SMOKE_TEST_ADMIN_USER_EMAIL }}
-                SMOKE_TEST_CUSTOMER_USER: ${{ secrets.SMOKE_TEST_CUSTOMER_USER }}
-                SMOKE_TEST_CUSTOMER_PASSWORD: ${{ secrets.SMOKE_TEST_CUSTOMER_PASSWORD }}
-                WC_E2E_SCREENSHOTS: 1
-                E2E_RETEST: 1
-                E2E_RETRY_TIMES: 0
-                E2E_SLACK_TOKEN: ${{ secrets.SMOKE_TEST_SLACK_TOKEN }}
-                E2E_SLACK_CHANNEL: ${{ secrets.SMOKE_TEST_SLACK_CHANNEL }}
-                UPDATE_WC: 1
-                DEFAULT_TIMEOUT_OVERRIDE: 120000
+                  BASE_URL: ${{ secrets.SMOKE_TEST_PERF_URL }}/
+                  ADMIN_USER: ${{ secrets.SMOKE_TEST_PERF_ADMIN_USER }}
+                  ADMIN_PASSWORD: ${{ secrets.SMOKE_TEST_PERF_ADMIN_PASSWORD }}
+                  CUSTOMER_USER: ${{ secrets.SMOKE_TEST_PERF_ADMIN_USER }}
+                  CUSTOMER_PASSWORD: ${{ secrets.SMOKE_TEST_PERF_ADMIN_PASSWORD }}
```

Comment:
If I retained the `CUSTOMER_USER` and `CUSTOMER_PASSWORD` values from `trunk`, `global-setup.js` would fail on the _"Trying to log-in as customer..."_ part.

It seems that `trunk` doesn't have the correct customer credentials for the daily perf test site. So I used the same username and password of the admin user instead.